src/tests/ftest/nvme/pool_extend.py

```diff
@@ -45,7 +45,7 @@ def run_nvme_pool_extend(self, num_pool, oclass=None):
             "test_servers", "server_partition", "server_reservation", "/run/extra_servers/*")
         ior_test_sequence = self.params.get("ior_test_sequence", '/run/ior/iorflags/*')
 
-        total_servers = len(self.hostlist_servers) * 2
+        total_servers = self.server_managers[0].engines()
```

Comment:
FYI this is a property so this works too
```suggestion
        total_servers = self.server_managers[0].engines
```